packages/grid/x-grid/src/tests/layout.XGrid.test.tsx

```diff
@@ -42,7 +42,7 @@ describe('<XGrid /> - Layout', () => {
         </div>,
       );
       expect(ref.current).to.be.instanceof(window.HTMLDivElement);
-      expect(ref.current).to.equal(container.firstChild.firstChild.firstChild);
+      expect(ref.current).to.equal(container.firstChild.firstChild);
```

Comment:
It should really be:
```suggestion
      expect(ref.current).to.equal(container.firstChild);
```

but at least it's a step in the right direction. I didn't look into fixing it completely.